Yelp/YelpClient.swift

```diff
@@ -84,4 +84,26 @@ class YelpClient: BDBOAuth1RequestOperationManager {
                 completion(nil, error)
         })!
     }
+    
+    func searchWithTermAndLocation(term: String, location: String, completion: ([Business]!, NSError!) -> Void) -> AFHTTPRequestOperation {
+        // For additional parameters, see http://www.yelp.com/developers/documentation/v2/search_api
+        
+        // Default the location to San Francisco
+        var parameters: [String : AnyObject] = ["term": term]
+                
+        //change ll to location (ll return nil)
+        parameters["location"] = location;
+
```

Comment:
Note that the reason we used "ll" was that causes the results to return a distance from the restaurant. I believe that adding "location" doesn't return a distance. I could be wrong there, but that's what I remember right now.